plugins/services/src/js/utils/__tests__/TaskTableUtil-test.js

```diff
@@ -52,7 +54,7 @@ describe('TaskTableUtil', function () {
       }
     });
 
-    this.sortFunction = TaskTableUtil.getSortFunction('name');
+    this.getComparator = TaskTableUtil.getSortFunction('name');
```

Comment:
How about we leave this file as is AND create a JIRA ticket for renaming the comparator's? We should also be mindful that as we scale, it will be more difficult to enforce "soft" rules like this. We want to delegate as much as possible to the linter and outside of that we will have to pick our battles, keeping in mind that we also want to move fast and with community contribution. If something is easier to reason about let's use it and count it as an improvement.